files/setup_freeipa.sh

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# This script is called by the freeipa-enroll service.
+
+# These variables must be set before execution.  They will
+# be loaded from a file installed by cloud-init:
+# /var/lib/cloud/instance/freeipa-creds.sh
+# ADMIN_PW: The password for the IPA server's Kerberos admin role
+# HOSTNAME: The hostname of this IPA client (e.g. client.example.com)
+# REALM: The realm for the IPA server (e.g. EXAMPLE.COM)
+
+# shellcheck disable=SC1091
```

Comment:
It would be nice to briefly explain here why you are disabling that particular shellcheck rule

Reply:
See commit ea1d82d

Reply:
👍